portfolio/pom.xml

```diff
@@ -23,6 +23,16 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
```

Comment:
Prefer to have same indentation for the entire file. For example Line#27 using 2 spaces but Line#32 has 4 spaces.